tracer/src/Datadog.Trace/Telemetry/Collectors/ConfigurationTelemetryCollector.cs

```diff
@@ -142,6 +142,7 @@ public ICollection<TelemetryValue> GetConfigurationData()
                 new(ConfigTelemetryData.ActivityListenerEnabled, value: settings.IsActivityListenerEnabled),
                 new(ConfigTelemetryData.ProfilerLoaded, value: _profiler?.Status.IsProfilerReady),
                 new(ConfigTelemetryData.CodeHotspotsEnabled, value: _profiler?.ContextTracker.IsEnabled),
+                new(ConfigTelemetryData.StatsComputationEnabled, value: settings.StatsComputationEnabled),
```

Comment:
Should we record the send interval too?

Reply:
Sorry for the non-response! For now, I'd prefer not to because this value should rarely ever be set by the customer and we don't offer a toggle for it in the Tracer right now